#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: #327025 C locale has bad default for _NL_TIME_FIRST_WEEKDAY
# DP: Related bugs: #327025  BZ181
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Petter Reinholdtsen
# DP: Upstream status: BZ181
# DP: Date: 2006-01-10

Index: locale/C-time.c
===================================================================
--- locale/C-time.c.orig
+++ locale/C-time.c
@@ -136,7 +136,7 @@
     { .string = "\7" },
     { .word = 19971130 },
     { .string = "\4" },
-    { .string = "\7" },
+    { .string = "\1" },
     { .string = "\1" },
     { .string = "\1" },
     { .string = "" },
